src/util/index.ts

```diff
@@ -0,0 +1,7 @@
+export async function hasError(resp: any) {
+  return !!resp.data._ERROR_MESSAGE_ || !!resp.data._ERROR_MESSAGE_LIST_ || resp.data.error;
+}
+
+export async function isError(resp: any): Promise<boolean> {
```

Comment:
I think we should not return Promise

Reply:
Made the function async and returning a boolean value.